cannon/README.md

```diff
@@ -39,6 +39,7 @@ make cannon
 # Note:
 #  - The L2 RPC is an archive L2 node on OP MAINNET.
 #  - The L1 RPC is a non-archive RPC, also change `--l1.rpckind` to reflect the correct L1 RPC type.
+#  - The network flag is only suitable for specific networks. If you are running on the devnet, please use '--l2.genesis'.
```

Comment:
I think this is a useful change but would be great to clarify the comment a little bit. Which networks support the `--network` flag? If using `--l2.genesis`, what should the input be?